modules/bazel-diff/7.2.3/source.json

```diff
@@ -0,0 +1,4 @@
+{
+    "url": "https://github.com/Tinder/bazel-diff/archive/refs/tags/7.2.3.tar.gz",
```

Comment:
Could you create a stable release archive?

Reply:
Done